easybuild/easyconfigs/s/sympy/sympy-0.7.6.1-intel-2015b-Python-2.7.11.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'sympy'
+version = '0.7.6.1'
+
+homepage = 'http://sympy.org/'
+description = """SymPy is a Python library for symbolic mathematics. It aims to
+ become a full-featured computer algebra system (CAS) while keeping the code as
+ simple as possible in order to be comprehensible and easily extensible. SymPy
+ is written entirely in Python and does not require any external libraries."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['sympy-0.7.6_fix-numpy-include.patch']
+
+python = 'Python'
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:-1])
+
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [(python, pyver)]
+
+runtest = 'python setup.py test'
```

Comment:
no, there is no default currently implemented by `PythonPackage`, it just provides support to specify a test command and set things up so the tests can run...

not sure if this is common enough to be the default